src/cascadia/TerminalSettingsEditor/Resources/en-US/Resources.resw

```diff
@@ -775,4 +772,19 @@
     <value>Rename "{}"</value>
     <comment>Use '{}' to denote placement for the color scheme name to be renamed</comment>
   </data>
+  <data name="ColorScheme_DeleteButton.[using:Windows.UI.Xaml.Controls]ToolTipService.ToolTip" xml:space="preserve">
+    <value>Delete the current color scheme.</value>
+  </data>
+  <data name="ColorScheme_Rename.[using:Windows.UI.Xaml.Controls]ToolTipService.ToolTip" xml:space="preserve">
+    <value>Rename</value>
+  </data>
+  <data name="Profile_DeleteButtonDisclaimerInBox" xml:space="preserve">
+    <value>This profile cannot be deleted because it is shipped in-box.</value>
+  </data>
+  <data name="Profile_DeleteButtonDisclaimerDynamic" xml:space="preserve">
+    <value>This profile cannot be deleted because it is a dynamic profile.</value>

```

Comment:
Wording

Reply:
I think maybe "auto-generated" could be a better phrase here.

Reply:
"This profile cannot be deleted because it is automatically generated." ?